### DISCUSSION TITLE
Appending model to an array causing duplicates in the blade

### DISCUSSION THREAD

**Discussion by bradleyking-dev** [livewire/livewire#3024] [7. Bugs (archived)]
When appending models to an array the translation to the blade is getting duplicated.
So adding a new model with the click of a button, then a function appends a new model to the array.
On the blade, it is picking up the array version and the object version of the same model, which is causing duplicates and issues with keying.

If I DD the array in the function it shows the correct amount of items in the array.
If I DD the array in the blade it shows twice as much in the array.

![Screenshot 2021-05-05 at 10 25 50](https://user-images.githubusercontent.com/57000861/117121263-45064e80-ad8c-11eb-96d0-d7244c289222.png)

<img width="580" alt="Screenshot 2021-05-05 at 10 21 13" src="https://user-images.githubusercontent.com/57000861/117121848-15a41180-ad8d-11eb-9bf6-629bfa80958d.png">
### Stripped-down, copy-pastable code snippets

**Context**
- Livewire version: 2.4.4
- Laravel version: 8.4
- Alpine version: 2.7.3
- Browser: Chrome

**Comment by calebporzio** [comment]
Hi there, you will need to provide better code examples for others to help you.

Please try to recreate this issue in the simplest form possible. Ideally with no domain specific classes or models.

Then paste the actual code here rather than screenshots.

There currently isn't enough info for others to pitch in. Thanks!
